quill-core/src/main/scala/io/getquill/context/ActionMacro.scala

```diff
@@ -100,7 +100,7 @@ class ActionMacro(val c: MacroContext)
     q"""
       expanded.ast match {
         case io.getquill.ast.Returning(_, _, io.getquill.ast.Property(_, property)) => 
-          property
+          expanded.naming.column(property)
```

Comment:
Sorry for late response, however I couldn't find there any logic for applying NamingStrategy in that code. It seemed like that piece is responsible for renaming columns meaning - customizing column names `querySchema[Circle]("circle_table", _.radius -> "radius_column")`
However I found that this https://github.com/getquill/quill/blob/master/quill-sql/src/main/scala/io/getquill/context/sql/idiom/SqlIdiom.scala#L282 and https://github.com/getquill/quill/blob/master/quill-sql/src/main/scala/io/getquill/context/sql/idiom/SqlIdiom.scala#L305 could be the case. Please correct me if I'm wrong.